proto/warden/sched/module/module.proto

```
@@ -0,0 +1,12 @@
+syntax = "proto3";
+package warden.sched.module;
+
+import "cosmos/app/v1alpha1/module.proto";
```

Buf (1.47.2)

import "cosmos/app/v1alpha1/module.proto": file does not exist

(COMPILE)